### PR TITLE
feat:added sponsor logos

### PR DESCRIPTION
**Pull request by aryab2003**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [ ] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.
- [ ] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: config/content/Sponsor.js [hunk 1]
```diff
@@ -126,12 +166,20 @@ export default {
     },
   ],
   deltaFirst: [
+    {
+      src: 'https://res.cloudinary.com/dkpdi1wek/image/upload/v1708785958/download_craeqs.png',
+      name: 'NordVPN',
+      link: 'https://nordvpn.com/',
+      alt: 'nordvpn LOGO',
+      // width: '20%',
+    },
```
Comment (ashutosh-rath02) [inline config/content/Sponsor.js:175]: Remove this naming system. Name according to sponsor category.

File: config/content/Sponsor.js [hunk 2]
```diff
@@ -84,6 +117,13 @@ export default {
     // },
   ],
   gamma: [
+    {
+      src: 'https://res.cloudinary.com/dkpdi1wek/image/upload/v1708784952/404_ww9ttr.png',
+      name: 'auth0',
+      link: 'https://auth0.com/',
+      alt: 'Auth0 LOGO',
+      // width: '20%',
+    },
```
Comment (ashutosh-rath02) [inline config/content/Sponsor.js:126]: Why is Auth0 placed twice?

File: config/content/Sponsor.js [hunk 3]
```diff
@@ -66,6 +67,38 @@ export default {
...
+
+    {
+      src: 'https://res.cloudinary.com/dkpdi1wek/image/upload/v1708787057/download_6_sop5ix.png',
+      name: 'auth0',
+      link: 'https://auth0.com/',
+      alt: 'Auth0 LOGO',
+      // width: '20%',
+    },
+
```
Comment (ashutosh-rath02) [inline config/content/Sponsor.js:101]: Where is ''flow"", its data is available in devfolio, add it.

**Review by ashutosh-rath02 — CHANGES_REQUESTED** [review]

File: config/content/Sponsor.js (hunk 3 above)
Comment (aryab2003) [reply config/content/Sponsor.js:101]: done

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]
@aryab2003 both tiers in green colour hence not looking good.

**Pull request closed without merging**